lib/liveman_web/router.ex

```diff
@@ -9,15 +9,22 @@ defmodule LivemanWeb.Router do
     plug :put_secure_browser_headers
   end
 
+  pipeline :require_authenticated_user do
+    plug LivemanWeb.Plugs.Auth
+  end
+
   # coveralls-ignore-start
   pipeline :api do
     plug :accepts, ["json"]
+  end
 
-    scope "/v1", LivemanWeb, as: :api_v1 do
-      get "/me", V1.UserController, :show
-      get "/surveys", V1.SurveyController, :index
-      post "/responses", V1.AnswerController, :create
-    end
+  scope "/v1", LivemanWeb, as: :api_v1 do
+    pipe_through :api
+    get "/surveys", V1.SurveyController, :index
+
+    pipe_through :require_authenticated_user
```

Comment:
I didn't know we can have our pipe_throughs like this in the same scope. Nice 👍

Reply:
IMO It's not clear

Reply:
@andyduong1920 For this case it looks ok to me as we don't have to write many `scope`. I checked in the compass we don't have a convention yet. I will create an RFC issue in the compass.

Reply:
Yeah, agree better to have a convention for that, for me I prefer another approach so it's easy to maintenance and understand

Reply:
Please send a new PR to update if there is anything changed after the convention is set 👍

Reply:
I created a RFC issue .I will update it based on of the RFC output https://github.com/nimblehq/compass/issues/665